frontend/src/components/annotation/audio_annotation/audio_annotation_core.vue

```diff
@@ -1,63 +1,17 @@
 <template>
 <div>
-  <div style="position: relative">
```

Comment:
Removing `auido_toolbar` and `audio_sidebar` from `audio_annotation_core` to `toolbar_factory` and `sidebar_factory` to enable audio annotation for compound files